admin/pg12/do-pg_upgrade.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -e
+shopt -s extglob
+
+PGDATA=/var/lib/postgresql/data
+PGDATA_OLD="$PGDATA"/9.5
+PGDATA_NEW="$PGDATA"/12
+PGAMQP_DIR=/tmp/pg_amqp
+
+function cleanup() {
+	if [[ -d "$PGDATA_NEW" ]]; then
```

Comment:
I don' think this is necessary. ``-f`` ignores if the directory exists and still outputs exit code ``0``.

Reply:
Echo about cleanup in https://github.com/metabrainz/musicbrainz-docker/pull/146/commits/8776cf29bdd4b2c2421e37505cc13fe51ac30fe8